tests/Homely.AspNetCore.Mvc.Helpers.Tests/TestControllerTests/ValidationErrorTests.cs

```diff
@@ -1,22 +1,33 @@
 ﻿using Homely.AspNetCore.Mvc.Helpers.Models;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Testing;
 using Shouldly;
 using System;
 using System.Collections.Generic;
 using System.Net;
 using System.Threading.Tasks;
+using TestWebApplication;
 using Xunit;
 
 namespace Homely.AspNetCore.Mvc.Helpers.Tests.TestControllerTests
 {
-    public class ValidationErrorTests : TestSetup
+    public class ValidationErrorTests : IClassFixture<TestFixture>
     {
+        private readonly TestFixture _factory;
+
+        public ValidationErrorTests(TestFixture factory)
+        {
+            _factory = factory ?? throw new ArgumentNullException(nameof(factory));
+        }
+
         // Controller manually threw a ValidationException.
         [Fact]
         public async Task GivenAGetRequestWhichManuallyThrowsAValidationError_ValidationError_ReturnsAnHttp400()
         {
             // Arrange.
+            var client = _factory.CreateClient();
+
             var error = new ValidationProblemDetails
             {
                 Type = "https://httpstatuses.com/400",

```

Comment:
why is this one still the old type?

Reply:
that shouldn't be. BRB.

Reply:
Phew! found it.

✅  mapping is [defined in some `INTERNAL` aspnetcore code](https://github.com/dotnet/aspnetcore/blob/00846a232c58db75c8477c9288af23508e055711/src/Mvc/Mvc.Core/src/DependencyInjection/ApiBehaviorOptionsSetup.cs#L57).
- PD's are also all internal and created there ... so the `Type` value is always handled 'internally'.

so manual creation of PD's is a very custom scenario and therefore can't leverage the smarts from aspnetcore.

The other idea was to use _their_ code to create the PD but again, [that code is internal](https://github.com/dotnet/aspnetcore/blob/2e4274cb67c049055e321c18cc9e64562da52dcf/src/Mvc/Mvc.Core/src/Infrastructure/DefaultProblemDetailsFactory.cs#L12).

so going to leave this value `null`.